packages/v4/src/content/accessibility/badge/badge.md

```diff
@@ -10,3 +10,7 @@ A **badge** is used to annotate other information like a label or an object name
 **Screen reader users** should be able to have a screen reader describe the contents of and purpose of a badge when it pops up, but not focus on it otherwise.
 
 No props need to be added or modified for badge accessibility.
+
+As a caution, badges can be styled using a variety of colors. Relying on color alone to communicate information causes barriers to 
```

Comment:
I wonder if it's worth making a simple suggestion of what to do instead, in case users were planning to do this "rely on color alone"...

Since the badge doesn't get an accessible name and isn't focusable, maybe we can just suggest to ensure there is adequate contextual information provided in the surrounding UI parts to convey the unit of measure for the number in the badge in case it isn't obvious?

Or maybe could just mention to consider using [label with icon](https://patternfly-org-pr-2893-v4.surge.sh/v4/components/label) if the badge count alone isn't intuitive enough?

Reply:
I agree with these above comments. When reading it, I also thought something more action oriented would be better like "Do not rely on color alone to communicate" because I think it's clearer for those skimming the docs. 

I think an example image could be powerful as well. Perhaps we could show badges or labels with just using color but through another lens (what it looks like when you have this eye condition, etc). I think Eric's idea of required labels being in red (but they all look the same to another user) would be really informative. I remember that was effective when we were talking about chart a11y.